src/client.rs

```diff
@@ -155,6 +157,13 @@ pub struct Client<T: Read + Write> {
     conn: Connection<T>,
 }
 
+impl<T: Read + Write> Client<T> {
+    /// Manually increment the current tag.
```

Comment:
Could you expand the docs a bit more here to indicate why someone may want to use this method?